<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parameters</class>
 <widget class="QWidget" name="parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QSpinBox" name="nbOfPtsToProject">
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDialogButtonBox" name="finishButtons">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Number of points to project at once</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Number of upsamplings</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="maxUpsampling">
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Size of grid for Morton Code</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="sizeOfMortonCodeGrid">
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>64</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
